### ISSUE TITLE
Buoy data issues

### ISSUE THREAD

**Issue by kkoch** [glos/myglos#218]
•	45027 is showing ‘thermistor string’ for the data instead of the expected ‘WaterTemperature @ depth’ so we aren’t seeing data for those fields in Portal

parameters: [
"WindDirection",
"WindSpeed",
"AirTemperature",
"Dew Point",
"Wind Gust",
"Air Pressure",
"Solar Radiation",
"Thermistor String Node 01 Depth In Water",
"Thermistor String Node 02 Depth In Water",
"Thermistor String Node 03 Depth In Water",
"Thermistor String Node 04 Depth In Water",
"Thermistor String Node 05 Depth In Water",
"Thermistor String Node 06 Depth In Water",
"Thermistor String Node 07 Depth In Water",
"Thermistor String Node 08 Depth In Water",
"Thermistor String Node 09 Depth In Water",
"Thermistor String Node 10 Depth In Water",
"WaterTemperature"
],

•	45028 does not show data for ‘Significant Wave Period’
•	45169 does not show data for ‘Air Pressure’
•	45170 does not show data for ‘SignificantWaveHeight’
•	GLERLWE2 was online but doesn’t seem to have data since Sunday, May 13 (maybe because there are parameters that are not showing data > dissolved oxygen, DO sat, conductivity, water temp, algae, chl, turbidity)

@gcutrell please check to be sure obs is okay for the above. If not, I will check with the appropriate PI (unless that is you and then you can check with yourself <grin>).

**Comment by kkoch** [comment]
@gcutrell > Few more:

- Looks like 45022 also has a parameter in obs for 'WaterTemperature' and 'WaterTemperature @   0 ft".  The '0 ft' one isn't displaying in the Portal and I tend to think it might not be a valid parameter, but please verify.
- 45161 not showing data for 'Air Temperature' and 'Water Temperature at Surface"
- 45176b not showing data for DO and DO-SAT
- 45168 not showing data for solar radiation

**Comment by gcutrell** [comment]
45027 - Working on glbuoys. Not sure where the error might be introduced. 
45028 - Shown on glbuoys. 
45169 - Shown on glbuoys.
45170 -  Shown on glbuoys.
GLERLWE2 - I'm guessing they were testing the buoy and forgot to stop data sharing. 
45022  - I don't see any current temperature at depth values in the database.
45161 - They are now sending water depth at 1 meter. They are also not sending air temperature so you could check with the PI about that. 
45176b - Looks like Ed just fixed it. 
45168 - Removed this off-season. Forgot to tell you.

**Comment by kkoch** [comment]
45027 is good (guessing the issue was resolved by the PI before Greg got a chance to dig into it); 45169, 45170, WE2, 45176b are okay as well). 

For the following I've commented out the parameter in GN so we won't see 'no results' in the portal. I double-checked on all of these to make sure they weren't showing for ndbc either, so we can assume the PI isn't submitting although I'm still puzzled why the php I use to look at obs (http://data.glos.us/portal/getObsCatalog.php?providers=glos) shows them when it hasn't in the past. >
-45022 water temp at 0 ft
-45028 wave period (note: Greg you said it was ok on glbuoys but I don't see it there either)
-45161 air temperature
-45168 solar radiation
